### ISSUE TITLE
Use system-installed NSS dependeny

### ISSUE THREAD

**Issue by larseggert** [mozilla/neqo#1711]
Rather than checking out and building our own copy of NSS during the build of neqo, we should detect and use a system-installed NSS. This will require some heavy surgery to `neqo-crypto/build.rs` but should really simplify things.

**Comment by larseggert** [comment]
https://github.com/mozilla/neqo/pull/1710#discussion_r1515305249

**Comment by larseggert** [comment]
@martinthomson at least the MacOS Homebrew package for NSS and the Ubuntu package for NSS do not include `mozpkix/pkixnss.h`, which neqo seems to depend on.

**Comment by martinthomson** [comment]
Yeah, we can probably just copy the thing that is currently generated into the code, annotate it as such, remove the binding, and move on with our lives.
